mappings/net/minecraft/entity/EntityType.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2245 net/minecraft/entity/EntityDispatcher
+CLASS net/minecraft/class_2245 net/minecraft/entity/EntityType
```

Comment:
`EntityManager` ftw

Reply:
We already discussed about this. pls stop

Reply:
but we have not decided on anything yet

Reply:
Yes, so it stays as `EntityType`. Its reverted on 1.8.9 too